Cargo.toml

```diff
@@ -16,7 +16,8 @@ path = "src/lib.rs"
 
 [features]
 default = [ "std" ]
-std = [ "serde/std" ]
+core = []
```

Comment:
Is there any reason ever _not_ to have `core` available? Does feature gating it make any sense? That feels like saying "oh, we might be missing basic arithmetic on some platforms" :laughing:.

Reply:
ah yeah true, core is always there, my bad, I was confused.

Reply:
pushed a fix